src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/BaseControllerTest.java

```diff
@@ -82,7 +84,7 @@ void doValidCaseDataSetUpForAdditionalHearing() throws IOException, URISyntaxExc
 
     protected CallbackRequest buildCallbackRequest() {
         Map<String, Object> caseData = new HashMap<>();
-        CaseDetails caseDetails = CaseDetails.builder().id(Long.valueOf(123)).data(caseData).build();
+        CaseDetails caseDetails = CaseDetails.builder().id(Long.valueOf(123)).caseTypeId(CASE_TYPE_ID_CONSENTED).data(caseData).build();
```

Comment:
This buildCallbackRequest is used in Contested tests in the controller. You should be able to add the caseTypeID into the data after we build a generic CaseDetails object

CallbackRequest callbackRequest = buildCallbackRequest();
callbackRequest.getCaseDetails().setCaseTypeId( xyz );

Reply:
agree with Harry, I think the cleaner solution is to have a consented and a contested buildCallbackRequest method that sets this as it will be a fairly common use case